### PR TITLE
Fixes some typos in docs

### PR DESCRIPTION
**Pull request by abhaykatheria**
Fixes some typos in Documentation

    These are the files that are changed:
       docs/adding-to-command-library.md
       docs/architecture.md
       docs/data-model.md
       docs/faq.md

Signed-off-by: abhay <abhay.katheria1998@gmail.com>

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # FAQ (Work in Progress)
 
 ## Why Tern?
-Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have an lwn subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
+Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have an own subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
```
Comment (nishakm) [inline docs/faq.md]: I meant LWN :)

**Comment by nishakm** [comment]
@abhaykatheria It looks to me that the changes don't just fix typos. Can you explain in your commit message what these changes fix?

**Comment by nishakm** [comment]
@abhaykatheria Will you be updating this PR?

**Comment by abhaykatheria** [comment]
@nishakm I will do it asap

**Comment by nishakm** [comment]
@abhaykatheria thanks for fixing this up! Could you be more descriptive in your commit message?
```
(A brief description of what was changed - typos, grammar?)
The following files were changed:
-docs/adding-to-command-library.md <-- I don't think this is necessary. git can show you what files where changed
-docs/architecture.md
-docs/data-model.md
-docs/faq.md
```

File: docs/faq.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # FAQ (Work in Progress)
 
 ## Why Tern?
-Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have an lwn subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
+Tern was created to help developers meet open source compliance requirements for containers. Open source software compliance is a hard problem in general but it gets harder with containers due to the ability to reuse diff filesystems. How those filesystems were created is still an ad hoc process. If you happen to have a lwn subscription you can read an article about it [here](https://lwn.net/Articles/752982/).
```
Comment (nishakm) [inline docs/faq.md:4]: While you are here, can you change `lwn` to `LWN`?

Comment (abhaykatheria) [reply docs/faq.md:4]: sure :)

**Pull request merged**